.mystools/cppcheck/config/suppressions.cfg

```diff
@@ -1,5 +1,4 @@
 toomanyconfigs
-missingInclude
-missingIncludeSystem
 ConfigurationNotChecked
 unmatchedSuppression
+unreadVariable:*/MyHwNRF5.cpp
```

Comment:
I had to add this because the error is in a macro so it throws off the in-line suppression logic of cppcheck

Reply:
Thanks for explaining why. Should the explanation be inline with the code instead of being in a comment in a review? Or would having inline comments clutter the code too much? We prefer having (doxygen)documentation inline, so maybe we should have cppcheck exceptions inline as well?

Reply:
I don't know if the cfg file supports comments. I have not been able to verify that.

Reply:
Seems like it does. From the cppcheck manual.
## Listing suppressions in a file
You can create a suppressions file. Example:
```
// suppress memleak and exceptNew errors in the file src/file1.cpp
memleak:src/file1.cpp
exceptNew:src/file1.cpp
```